ubiquibot-config-default.jsonc

```diff
@@ -1,7 +1,15 @@
 {
   "evm-network-id": 100,
-  "price-multiplier": 1.5,
-  "issue-creator-multiplier": 2000,
+  "price-multiplier": 1,
+  "issue-creator-multiplier": 1000,
+  "payment-permit-max-price": 9007199254740991 /* Number.MAX_SAFE_INTEGER */,
+  "max-concurrent-assigns": 9007199254740991 /* Number.MAX_SAFE_INTEGER */,
+  "assistive-pricing": false,
+  "disable-analytics": false,
+  "comment-incentives": false,
+  "register-wallet-with-verification": false,
+  "promotion-comment": "\n<h6>If you enjoy the DevPool experience, please follow <a href='https://github.com/ubiquity'>Ubiquity on GitHub</a> and star <a href='https://github.com/ubiquity/devpool-directory'>this repo</a> to show your support. It helps a lot!</h6>",
+  "default-labels": [],
```

Comment:
It could be nice to have everything labels related to be nested. For example:

```json
{
  "labels": {
    "default": [],
    "time": [
      {
        "name": "Time: <1 Hour",
        "weight": 0.125,
        "value": 3600
      },
      {
        "name": "Time: <1 Day",
        "weight": 1,
        "value": 86400
      },
      {
        "name": "Time: <1 Week",
        "weight": 2,
        "value": 604800
      },
      {
        "name": "Time: <2 Weeks",
        "weight": 3,
        "value": 1209600
      },
      {
        "name": "Time: <1 Month",
        "weight": 4,
        "value": 2592000
      }
    ],
    "priority": [
      {
        "name": "Priority: 0 (Normal)",
        "weight": 1
      },
      {
        "name": "Priority: 1 (Medium)",
        "weight": 2
      },
      {
        "name": "Priority: 2 (High)",
        "weight": 3
      },
      {
        "name": "Priority: 3 (Urgent)",
        "weight": 4
      },
      {
        "name": "Priority: 4 (Emergency)",
        "weight": 5
      }
    ]
  }
}
```

Reply:
Is `labels.time.weight` necessary if we already have `labels.time.value`?

Reply:
Perhaps we should remove `weight` for both `time` and `priority`, and then only use `value`.

```json
{
  "labels": {
    "default": [],
    "time": [
      {
        "name": "Time: <1 Hour",
        "value": 3600
      },
      {
        "name": "Time: <1 Day",
        "value": 86400
      },
      {
        "name": "Time: <1 Week",
        "value": 604800
      },
      {
        "name": "Time: <2 Weeks",
        "value": 1209600
      },
      {
        "name": "Time: <1 Month",
        "value": 2592000
      }
    ],
    "priority": [
      {
        "name": "Priority: 0 (Normal)",
        "value": 1
      },
      {
        "name": "Priority: 1 (Medium)",
        "value": 2
      },
      {
        "name": "Priority: 2 (High)",
        "value": 3
      },
      {
        "name": "Priority: 3 (Urgent)",
        "value": 4
      },
      {
        "name": "Priority: 4 (Emergency)",
        "value": 5
      }
    ]
  }
}
```

Reply:
> Is `labels.time.weight` necessary if we already have `labels.time.value`?

As far as I understand both are necessary. `labels.time.weight` is used for pricing while `labels.time.value` is used for issues' time boundaries

Reply:
What are "time boundaries"?

We only had ranges for prices in the past but those have since been deprecated.

Reply:
> What are "time boundaries"?

"Time boundary" is the amount of seconds for solving an issue. 

Check this config:
```
{
        "name": "Time: <1 Hour",
        "weight": 0.125,
        "value": 3600
}
```

Issue's price label is calculated this way: `Pricing = timeLabel.weight * 100 * priceMultiplier`
When a bounty hunter self assigns an issue then he has `timeLabel.value` seconds to solve an issue

So both `timeLabel.weight` and `timeLabel.value` are necessary

Reply:
There's a task open for deriving the amount of time using the Vercel `ms` library.

Reply:
Then we should wait for https://github.com/ubiquity/ubiquibot/issues/431 and https://github.com/ubiquity/ubiquibot/issues/495

Then we could indeed remove `timeLabel.weight` and `timeLabel.value`